File: gcd.js
function gcd(a, b) {




    if (b == 0) {
        return a;
    }

    var aprime = a % b;

    return gcd(b, aprime)



}

function lcm(a, b) {

    return (a * b) / gcd(a, b)



}


console.log("gcd", lcm(75, 15))

File: algorithms specilization/Algorithmic Toolbox/week2_algorithmic_warmup/3_greatest_common_divisor/GCD.java
import java.util.*;

public class GCD {
public static int gcd(int a,int  b) {




    if (b == 0) {
        return a;
    }

    int aprime = a % b;

    return gcd(b, aprime);



}

  public static void main(String args[]) {
    Scanner scanner = new Scanner(System.in);
    int a = scanner.nextInt();
    int b = scanner.nextInt();

    System.out.println(gcd(a, b));
  }
}
